synapse/models/economic.py

```diff
@@ -237,108 +238,164 @@
                 ('price', ('econ:price', {}), {
                     'doc': 'The total cost of this receipt line item.'}),
 
-                # FIXME econ:valuable
                 ('product', ('biz:product', {}), {
                     'doc': 'The product being being purchased in this line item.'}),
             )),
 
-            ('econ:acct:payment', {}, (
+            ('econ:cash:deposit', {}, (
 
-                ('id', ('meta:id', {}), {
-                    'prevnames': ('txnid',),
-                    'doc': 'A payment processor specific transaction ID.'}),
+                ('time', ('time', {}), {
+                    'doc': 'The time the cash was deposited.'}),
 
-                ('fee', ('econ:price', {}), {
-                    'doc': 'The transaction fee paid by the recipient to the payment processor.'}),
+                ('actor', ('entity:actor', {}), {
+                    'doc': 'The entity who deposited the cash.'}),
```

Comment:
This seems like it should be "The entity which deposited" rather than "who", (there is a similar docstring for withdrawals that has the same thing)